### ISSUE TITLE
Deploying smart contract timing out

### ISSUE THREAD

**Issue by ganesansays** [monax/cli#1248]
Please include in your bug report:

- [ ] `docker version`
v1.13.1
- [ ] `eris version`
Eris CLI Version: 0.12.0
- [ ] `go version` (if applicable)
- [ ] `docker-machine version` (if applicable / !linux)
docker-machine version 0.9.0, build 15fd4c7
- [ ] operating system details (osx/windows/linux)
ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-20160627
- [ ] what you expected to happen
Contract gets deployed to the chain and test succeed
- [ ] what actually happened
```
Performing action. This can sometimes take a wee while
Executing Job                                 defaultAddr
Executing Job                                 setStorageBase
Executing Job                                 deployStorageK
Response                                  abi=[{"constant":false,"inputs":[{"name":"x","type":"uint256"}],"name":"set","outputs":[],"type":"function"},{"constant":true,"inputs":[],"name":"get","outputs":[{"name":"retVal","type":"uint256"}],"type":"function"}]
                                          bin=606060405260978060106000396000f360606040526000357c01000000000000000000000000000000000000000000000000000000009004806360fe47b11460415780636d4ce63c14605757603f565b005b605560048080359060200190919050506078565b005b606260048050506086565b6040518082815260200191505060405180910390f35b806000600050819055505b50565b600060006000505490506094565b9056
                                         name=IdisContractsFTW
Deploying Contract                       name=IdisContractsFTW
Encountered error waiting for event: timed out waiting for event

                                         addr=0F6707DB409BA3BD51E6E0489D9F7E2B898327C4
Executing Job                                 setStorage
Deprecation Warning: The use of the 'data' field to specify the name of the contract function has been deprecated. Please update your epm jobs file to utilize a combination of 'function' and 'data' fields instead. See documentation for further details.
Encountered error waiting for event: timed out waiting for event

Executing Job                                 queryStorage
Deprecation Warning: The use of the 'data' field to specify the name of the contract function has been deprecated. Please update your epm jobs file to utilize a combination of 'function' and 'data' fields instead. See documentation for further details.
Return Value                                  0
Executing Job                                 assertStorage
Assertion Failed                              0 == 5
assertion failed

Could not perform pkg action service: Could not perform pkg action: Container interactive-6e8036e7-d2bd-4199-95b5-f2b9c55c2716 exited with status 1

```
- [ ] steps to reproduction
```
-- Start Blockchain
eris chains make --account-types=Root:1,Validator:3 myadvchain

docker-machine create --driver amazonec2 root-000
eris init --machine root-000 advchain --yes
eris chains start myadvchain --init-dir myadvchain/myadvchain_root_000 --machine "root-000" myadvchain

for i in `seq 0 2`
do
  docker-machine create --driver amazonec2 "validator-00$i"
done

--Make sure the ips of the docker machines are added to the seeds

for i in `seq 0 2`
do
  eris init --machine "validator-00$i" advchain --yes
done

for i in `seq 0 2`
do
  eris services start logrotate --machine "validator-00$i"
done

for i in `seq 0 2`
do
  eris chains start myadvchain --init-dir myadvchain/"myadvchain_validator_00$i" --machine "validator-00$i" myadvchain
done

--deloy contracts
addr=$(cat ~/.eris/chains/myadvchain/addresses.csv | grep myadvchain_root_000 | cut -d ',' -f 1)
echo $addr
eris pkgs do --chain myadvchain --address $addr --machine root-000
```
- [ ] `additional information`
Blockchain is successfully created. Blocks are getting generated without any issue.
I am using the IdisContractsFTW part of the "getting started" tutorial
Same contract without any modification gets deployed to simplechain successfully

If issue is a feature request, tell us why this feature is useful.

**Comment by ganesansays** [comment]
**Additional information** 
1) EC2 Security Group is open to the world both inbound and outbound.
 2) I have tried hosting the root node in my local machine which had the same problem and that is the reason I tried deploying that also to EC2

**Comment by ganesansays** [comment]
@zramsay, appreciate your help.

**Comment by zramsay** [comment]
did you update the `seeds` field in the `config.toml` in the second node to point to the first node's IP?

can you confirm the chain is running with `eris chains logs myadvchain`? You'll see the blocks increasing after running the command a couple times.

**Comment by ganesansays** [comment]
@zramsay, I update all the config.toml to the same set of IPs of the three validators comma separated. As I have already mentioned, I was able to see blocks getting created in the logs.

**Comment by zramsay** [comment]
@ganesansays we have released version 0.16, can you reproduce on this version?

**Comment by zramsay** [comment]
closing for now. Note that the bug underlying this may have had to do with the way we implemented the `--machine` flag, which will be deprecated in the 0.17.0 release.
